### PR TITLE
[issue-451] use uritools.isabsuri for namespace validation

### PR DESCRIPTION
**Pull request by armintaenzertng**
part of #451

Signed-off-by: Armin Tänzer <armin.taenzer@tngtech.com>

### CODE REVIEW

File: spdx/parsers/validations.py [hunk 1]
```diff
@@ -151,11 +152,7 @@ def validate_doc_spdx_id(value, optional=False):
...
         return optional
-    elif (
-        value.startswith("http://")
-        or value.startswith("https://")
-        or value.startswith("ftp://")
-    ) and ("#" not in value):
+    elif uritools.isabsuri(value) and ("#" not in value):
         return True
     else:
         return False
```
Comment (meretp) [inline spdx/parsers/validations.py:158]:
```suggestion
    return uritools.isabsuri(value) and ("#" not in value)
```

I think we could shorten this a bit.

**Review by meretp — CHANGES_REQUESTED** [review]
Thanks! One small remark from my side.

File: spdx/parsers/validations.py (hunk 1 above)
Comment (armintaenzertng) [reply spdx/parsers/validations.py]: Good point. I was reluctant to touch the structure at first, but going through the whole file erasing lines was quite satisfying :D

**Review by meretp — APPROVED** [review]
Looks good to me, I didn't intend to change the whole file, but this clearly improves the readability, thank you!

**Pull request merged**